enclave-runtime/Cargo.lock

```diff
@@ -1034,6 +1034,15 @@ dependencies = [
  "sp-version",
 ]
 
+[[package]]
+name = "frame-system-rpc-runtime-api"
+version = "4.0.0-dev"
+source = "git+https://github.com/paritytech/substrate.git?branch=polkadot-v0.9.28#ce10b9f29353e89fc3e59d447041bb29622def3f"
+dependencies = [
+ "parity-scale-codec",
+ "sp-api",
+]
+
```

Comment:
Do you know why we suddenly have this dependency?

Reply:
Because of the constant...

Reply:
This should not be the case as the settings toml is empty, but I could find it in the other PRs too for whatever reason. So it should not be related to your change.

Reply:
Yes, we forgot to commit the Cargo.toml at some point, so these changes are now emerging in all the open PRs we have..